_maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm

```diff
@@ -32,7 +32,7 @@
 /obj/structure/stone_tile/slab,
 /obj/item/storage/bag/quiver,
 /turf/open/misc/asteroid/basalt/lava_land_surface,
-/area/template_noop)
+/area/ruin/unpowered/ash_walkers)
```

Comment:
Don't we have to leave it powered or else drying racks stop working?

Reply:
that's how the ashwalker base has always been, the drying rack already there I edited to just not use power

in reality they shouldn't use power at all but what can you do